packages/react-search-ui/src/containers/__tests__/ResultsPerPage.test.tsx

```diff
@@ -1,9 +1,7 @@
 import React from "react";
 import { shallow } from "enzyme";
-import {
-  ResultsPerPageContainer,
-  ResultsPerPageViewProps
-} from "../ResultsPerPage";
+import { ResultsPerPageContainer } from "../ResultsPerPage";
+import { ResultsPerPageViewProps } from "@elastic/react-search-ui-views";
```

Comment:
Glad you raised that question. Its something I want to address but will need a breaking change. Right now even before TS, `react-search-ui` was dependant on `react-search-ui-views` and because of this, ive had to place alot of types that should of been in `react-search-ui` to be in `react-search-ui-views` otherwise there will be a circular dep issue.

next couple of days I will go into more details about this change.